dao/authentication_dao_test.go

```diff
@@ -348,6 +348,7 @@ func TestAuthFromVault(t *testing.T) {
 		t.Errorf(`wrong type for the secret's data object. Want "map[string]interface{}", got "%s"'`, reflect.TypeOf(vaultData["data"]))
 	}
 	data["last_available_at"] = authentication.LastAvailableAt.Format(time.RFC3339Nano)
+	data["password"] = "my-password"
```

Comment:
... so that you can use it here without hard coding the value
```diff
- data["password"] = "my-password"
+ data["password"] = *authentication.Password
```